lib/src/components/movement_sensor/service.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:grpc/grpc.dart';
+
+import '../../gen/common/v1/common.pb.dart';
+import '../../gen/component/movementSensor/v1/movementSensor.pbgrpc.dart';
+import '../../resource/manager.dart';
+import '../../utils.dart';
+import 'movement_sensor.dart';
+
+class MovementSensorService extends MovementSensorServiceBase {
+  final ResourceManager _manager;
+
+  MovementSensorService(this._manager);
+
+  MovementSensor _fromManager(String name) {
+    try {
+      return _manager.getResource(MovementSensor.getResourceName(name));
+    } catch (e) {
+      throw (GrpcError.notFound(e.toString()));
+    }
+  }
+
+  @override
+  Future<DoCommandResponse> doCommand(ServiceCall call, DoCommandRequest request) async {
+    final MovementSensor movementSensor = _fromManager(request.name);
+    final result = await movementSensor.doCommand(request.command.toMap());
+    return DoCommandResponse(result: result.toStruct());
+  }
+
+  @override
+  Future<GetAccuracyResponse> getAccuracy(ServiceCall call, GetAccuracyRequest request) async {
+    final MovementSensor movementSensor = _fromManager(request.name);
+    final accuracyMm = await movementSensor.accuracy(extra: request.extra.toMap());
+    return GetAccuracyResponse(accuracyMm: accuracyMm);
+  }
+
+  @override
+  Future<GetAngularVelocityResponse> getAngularVelocity(ServiceCall call, GetAngularVelocityRequest request) async {
+    final MovementSensor movementSensor = _fromManager(request.name);
+    final Vector3 angularVelocity = await movementSensor.angularVelocity(extra: request.extra.toMap());
```

Comment:
Q: why explicitly add the type here (and elsewhere in this file)?

Reply:
For no good reason haha. I originally had them all defined without the type, but then thought it was more clear to add the type.

There was also one spot where the type inference didn't work in a different service because of an overloaded type name, but that isn't a reason to type everything strictly. 

I read over darts suggestions on annotating type, seems we ought to not annotate local variables unless we need to.

> Don’t annotate locals and generic invocations unless you need to.

https://dart.dev/effective-dart/design#types

I'll rework this, and then make sure other servers follow suit, i did this elsewhere.

Reply:
https://github.com/viamrobotics/viam-flutter-sdk/pull/36